src/infrastructure/NodeService.js

```diff
@@ -67,50 +67,55 @@ class NodeService {
 	 * @param {object} nodeInfo NodeInfoDTO.
 	 * @returns {object} readable NodeInfo.
 	 */
-	static formatNodeInfo = nodeInfo => ({
-		...nodeInfo,
-		nodePublicKey: nodeInfo.publicKey,
-		address: symbol.Address.createFromPublicKey(
-			nodeInfo.publicKey,
-			nodeInfo.networkIdentifier
-		).plain(),
-		rolesRaw: nodeInfo.roles,
-		roles: [1,4,5].includes(nodeInfo.roles) && nodeInfo.apiStatus?.isAvailable
-			? Constants.RoleType[nodeInfo.roles] + ' (light)'
-			: Constants.RoleType[nodeInfo.roles],
-		network: Constants.NetworkType[nodeInfo.networkIdentifier],
-		version: helper.formatNodeVersion(nodeInfo.version),
-		apiEndpoint:
-			2 === nodeInfo.roles ||
-			3 === nodeInfo.roles ||
-			6 === nodeInfo.roles ||
-			7 === nodeInfo.roles
-				? nodeInfo.apiStatus.restGatewayUrl
-				: Constants.Message.UNAVAILABLE
-	});
+	static formatNodeInfo = nodeInfo => {
+		const { hostname, port } = '' !== nodeInfo.endpoint
+			? new URL(nodeInfo.endpoint)
+			: { hostname: 'N/A', port: 'N/A' };
+
+		return {
+			finalizedEpoch: nodeInfo.finalizedEpoch,
+			finalizedHash: nodeInfo.finalizedHash,
+			finalizedHeight: nodeInfo.finalizedHeight,
+			finalizedPoint: nodeInfo.finalizedPoint,
+			height: nodeInfo.height,
+			mainPublicKey: nodeInfo.mainPublicKey,
+			isHealthy: nodeInfo.isHealthy,
+			restVersion: nodeInfo.restVersion,
+			isHttpsEnabled: nodeInfo.isSslEnabled,
+			friendlyName: nodeInfo.name,
+			geoLocation: nodeInfo.geoLocation,
+			host: hostname,
+			version: nodeInfo.version,
+			port,
+			address: symbol.Address.createFromPublicKey(
+				nodeInfo.mainPublicKey,
+				http.networkType
+			).plain(),
+			rolesRaw: nodeInfo.roles,
+			roles: [1,4,5].includes(nodeInfo.roles) && null != nodeInfo.restVersion
+				? Constants.RoleType[nodeInfo.roles] + ' (light)'
+				: Constants.RoleType[nodeInfo.roles],
+			network: Constants.NetworkType[http.networkType],
+			networkIdentifier: http.networkType,
+			apiEndpoint: null != nodeInfo.restVersion ? nodeInfo.endpoint : Constants.Message.UNAVAILABLE,
+			networkGenerationHashSeed: http.generationHash
+		};
+	};
 
 	/**
 	 * Get available node list from statistic service.
 	 * @returns {array} NodeInfo[]
 	 */
 	static getAvailableNodes = async () => {
 		try {
-			const nodePeers = await http.statisticServiceRestClient().getNodes();
+			const nodePeers = await NodeWatchService.getNodes();
```

Comment:
Also update the function doc

Reply:
Still missing updating this
``Get available node list from statistic service.``